extensionsGallery.json

```diff
@@ -3200,8 +3200,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.4.0",
-							"lastUpdated": "11/5/2020",
+							"version": "0.5.0",
+							"lastUpdated": "12/1/2020",
```

Comment:
You missed updating the link below

Reply:
I've sent you link to the correct file vsix via email.

Reply:
Updated. Thanks.